tools/downloader/converter.py

```diff
@@ -57,14 +57,24 @@ def prefixed_subprocess(prefix, args):
         return p.wait() == 0
 
 def convert_to_onnx(model, output_dir, args, stdout_prefix):
-    pytorch_converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
+    if model.pytorch_to_onnx_args:
+        converter = Path(__file__).absolute().parent / 'pytorch_to_onnx.py'
+        converter = [str(converter)]
+        converter_args = [string.Template(arg).substitute(conv_dir=output_dir / model.subdirectory,
+                                                            dl_dir=args.download_dir / model.subdirectory)
+                                for arg in model.pytorch_to_onnx_args]
+    if model.caffe2_to_onnx_args:
+        converter = ['-c', "import caffe2.python.onnx.bin.conversion;\
+                            import sys;\
+                            caffe2.python.onnx.bin.conversion.caffe2_to_onnx(sys.argv[1:])"]
```

Comment:
You don't actually have to pass `sys.argv` to the entrypoint - if you pass no arguments, it will use `sys.argv` by default.